src/config/build.js

```diff
@@ -6,8 +6,14 @@ const { BUILD_DIRECTORY_NAME } = require('../constants');
 const { combine } = require('../utils/structures');
 const { getProjectConfig } = require('./project');
 
+const PROJECT_TYPES_CONFIG = {
+  svelte: {
+    useCSSModules: false
```

Comment:
What's wrong with CSS Modules?

Reply:
Svelte does its own thing for CSS management (they're single-file components with self-contained styles, not imported modules).

If CSS Module transforms were enabled for Svelte projects, and you decided to extracted the resulting CSS into its own file (which is off by default), you'd end up with hashed classNames in the CSS, and no reference to them in the JS.

We might be able to improve on the current webpack config, by somehow determining that extracted styles came from a svelte component, and disable CSS Modules for those files alone. I'd like to see this, because we could then enable CSS Modules elsewhere in the project.